README

```diff
@@ -1,44 +1,25 @@
-approximate the physical force-free state with the same topology that you
-specified.
-
-FLUX consists of two parts: a relaxation engine, written in C, and a
-control interface in Perl.  The control interface is intended to be
-used with Perl Data Language (PDL).  If you do not have PDL on your
-system, you can get it for free from http://pdl.perl.org.  PDL
-compiles "out of the box" on most UNIX systems, and a "native"
-version exists for Microsoft Windows.
-
-The authors gratefully acknowledge support from NASA's Living With A
-Star and Solar/Heliospheric Physics programs, and from the Southwest
-Research Institute.
-
-All source files are copyright (C) 2004-2008 by the authors unless
-otherwise marked.  You may modify and/or make copies of this software
-under the terms of the Gnu Public License, version 2 (GPL v2).  You
-should have received a copy of the GPL v2 with this software; it is
-present in the file "COPYING".  If you do not have a copy you may
-obtain one online from "http://www.gnu.org/licenses/gpl-2.0.html" or
-by writing to the Free Software Foundation, 59 Temple Place, Suite
-330, Boston, MA 0211-1307 USA.
+FLUX is a prototype MHD solver that is currently able to find nonlinear force-free fields with an exactly specified topology in a quasi-static, infinitely conductive, low-beta medium.  Unlike conventional extrapolation codes, FLUX accepts as input a collection of field line shapes that do not necessarily have physical meaning but that do have the correct topology that you want to study.
+
+The FLUX engine relaxes the position of those field lines until they approximate the physical force-free state with the same topology that you specified.
+
+FLUX consists of two parts: a relaxation engine, written in C, and a control interface in Perl.  The control interface is intended to be used with Perl Data Language (PDL).  If you do not have PDL on your system, you can get it for free from http://pdl.perl.org.  PDL compiles "out of the box" on most UNIX systems, and a "native" version exists for Microsoft Windows.
+
+The authors gratefully acknowledge support from NASA's Living With A Star and Solar/Heliospheric Physics programs, and from the Southwest Research Institute.
+
+All source files are copyright (C) 2004-2023 by the authors unless otherwise marked.  You may modify and/or make copies of this software under the terms of the Gnu Public License, version 2 (GPL v2).  You should have received a copy of the GPL v2 with this software; it is present in the file "COPYING".  If you do not have a copy you may obtain one online from "http://www.gnu.org/licenses/gpl-2.0.html" or by writing to the Free Software Foundation, 59 Temple Place, Suite 330, Boston, MA 0211-1307 USA.
 
 THIS SOFTWARE COMES WITH NO WARRANTY OF ANY KIND.
 
 Authors:
-	Craig DeForest	 	deforest@boulder.swri.edu
-	Charles Kankelborg 	kankel@solar.physics.montana.edu
+	Craig DeForest		deforest@boulder.swri.edu
```

Comment:
Maybe delete all these email addresses? Replace them with the institutions they had when they were contributing.  Laurel was a CU grad student, Alisdair was a SwRI employee.
Something like:

> Laurel Rachmeler (then at University of Colorado Boulder & SwRI)
> Alisdair Davey (then at SwRI)